### DISCUSSION TITLE
Can a Path segment "Edge_format" only apply to the last folder?

### DISCUSSION THREAD

**Discussion by twvogelsang** [JanDeDobbeleer/oh-my-posh#4715] [Troubleshoot]
I'm trying to style my path segment to be similar to `agnoster_full` but to include a `folder_icon` for every intermediate folder..

I can do `folder_format": "\uf115 %s` to get 
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/830025/2159b165-e343-4198-89da-a0a79d12511d)

But I'd like to not have the folder icon next to home.

I can do `edge_format": "%s`, to get:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/830025/d39e8501-d04f-4a25-8825-17f3e67544f4)

which is cose. 

Is there some combination of `folder_fomat` and `edge_format` (or other options) that will display a folder_icon for every folder, including the current/last one, but NOT for home?

**Comment by JanDeDobbeleer** [comment]
@twvogelsang hmm, interesting. I'd argue we should leave the replaced locations alone but that's a challenge... There's no solution for this right now.

**Reply by JanDeDobbeleer** [reply]
@twvogelsang #4815
